agent/k8s/instana-k8s-mustgather.sh

```diff
@@ -6,10 +6,41 @@
 # This script collects data for the Instana Host Agent on Kubernetes / OpenShift
 #
 # Usage:
-#   ./instana-k8s-mustgather.sh
+#   ./instana-k8s-mustgather.sh [-n NAMESPACE]
+#
+# Options:
+#   -n NAMESPACE  Specify the Instana agent namespace (default: instana-agent)
+#   -h            Display this help message
 #
 ###############################################################################
 
+# Display usage information
+show_usage() {
+    echo "Usage: $0 [-n NAMESPACE]"
+    echo ""
+    echo "Options:"
+    echo "  -n NAMESPACE  Specify the Instana agent namespace (default: instana-agent)"
+    echo "  -h            Display this help message"
+    exit 1
```

Comment:
Maybe consider not exiting 1 here, since if someone were to run it with `-h` it would give the wrong impression that something went wrong.
Whilst if the option was indeed not recognized it should rightfully exit with 1.
Perhaps the exit is better placed inside each case with 0 for when the option is h and 1 `\?` case. WDTY?